src/utils/utils.jl

```diff
@@ -374,3 +374,18 @@ macro forward(sender, receiver, exclusions = Symbol[])
     end
     return esc(out)
 end
+
+# This function is used instead of cp given
+# https://github.com/JuliaLang/julia/issues/30723
+function copy_file(src::AbstractString, dst::AbstractString)
+    try
+        run(`cp -f $(src) $(dst)`)
+    catch e
+        if Sys.iswindows()
+            run(`cmd /c copy /Y $(src) $(dst)`)
```

Comment:
Sorry for the bad suggestion. Don't know why `cmd /c` is required.

Reply:
it seems that it depends on the shell interpreter on the windows machine.